### PR TITLE
add blockade to kubernetes for pkg/util

### PR DESCRIPTION
**Pull request by BenTheElder**
/cc @spiffxp @dims @liggitt 
one of the options for https://github.com/kubernetes/enhancements/issues/2420#issuecomment-785663043 4)

### CODE REVIEW

File: config/prow/plugins.yaml [hunk 1]
```diff
@@ -155,6 +155,11 @@ blockades:
   blockregexps:
   - ^examples/
   explanation: "examples/ has moved to https://github.com/kubernetes/examples/"
+- repos:
+  - kubernetes/kubernetes
+  blockregexps:
+  - ^pkg/util/[^/]+\.go$
+  explanation: "Utility code must be added to a subpackage like pkg/util/flag, not pkg/util directly. See: #49993"
```
Comment (BenTheElder) [inline config/prow/plugins.yaml:162]: https://github.com/kubernetes/kubernetes/issues/49993

Comment (BenTheElder) [reply config/prow/plugins.yaml:162]: er we probably wanted https://github.com/kubernetes/kubernetes/issues/49923, updating.

File: config/prow/plugins.yaml [hunk 2]
```diff
@@ -155,6 +155,11 @@ blockades:
   blockregexps:
   - ^examples/
   explanation: "examples/ has moved to https://github.com/kubernetes/examples/"
+- repos:
+  - kubernetes/kubernetes
+  blockregexps:
+  - ^pkg/util/[^/]+\.go$
+  explanation: "Utility code must be added to a subpackage like pkg/util/flag, not pkg/util directly. See: #49923"
```
Comment (BenTheElder) [inline config/prow/plugins.yaml:162]: https://github.com/kubernetes/kubernetes/issues/49923

**Review by spiffxp — APPROVED** [review]
/approve
/lgtm

**Pull request merged**